File: sms/src/main/java/dev/pitlor/sms/models/Mms.java
package dev.pitlor.sms.models;

import java.io.File;
import java.time.OffsetDateTime;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class Mms {
    File picture;
    String address;
    OffsetDateTime dateReceived;
    String subject;
    long threadId;
    String body;
}

File: sms/src/main/java/dev/pitlor/sms/models/Mms.kt
package dev.pitlor.sms.models

import java.io.File
import java.time.OffsetDateTime

data class Mms (
    val address: String,
    val dateReceived: OffsetDateTime,
    val threadId: Long,
    val picture: File? = null,
    val subject: String? = null,
    val body: String? = null
)